docs/shared/getting-started/intro.md

```diff
@@ -1,100 +1,87 @@
 # Intro to Nx
 
-Nx is a powerful open-source build system that provides tools and techniques for enhancing developer productivity, optimizing CI performance, and maintaining code quality. Find out more about [why you should use Nx](/getting-started/why-nx).
+Nx is a powerful open-source build system that provides tools and techniques for enhancing developer productivity, optimizing CI performance, and maintaining code quality.
 
-If instead you want to jump right into it, run the following command. It will guide you through the setup:
+## Core Features
 
-{% tabs %}
-{% tab label="npm" %}
+- **Run Tasks Efficiently**: Nx [runs tasks in parallel](/core-features/run-tasks) also considering dependencies between the tasks.
+- **Cache Locally & Remotely**: With [local](/core-features/cache-task-results) and [remote caching](/core-features/remote-cache), Nx prevents unnecessary re-runs of tasks, saving you valuable dev time.
+- **Automate Dependency Update**: if you leverage Nx plugins you gain additional features such as [code generation](/core-features/plugin-features/use-code-generators) and tools to [automatically upgrade](core-features/automate-updating-dependencies) your codebase and dependencies.
+- **Make it Your Own**: Nx is highly customizable and extensible. Fine-tune it by [create your own plugins](/extending-nx/intro/getting-started) and optionally even [share them with the community](/extending-nx/tutorials/publish-plugin#publish-your-nx-plugin).
 
-```shell
-npx create-nx-workspace
-```
+<!-- - **Monorepo and Single Projects**: Nx supports both, monorepos as well as single-project (standalone) workspaces. -->
 
-{% /tab %}
-{% tab label="yarn" %}
-
-```shell
-npx create-nx-workspace --pm yarn
-```
+Find out more about [why you should use Nx](/getting-started/why-nx) or browse our [core features](/core-features) section.
 
-{% /tab %}
-{% tab label="pnpm" %}
+## Try Yourself!
```

Comment:
```suggestion
## Try Nx Yourself!
```

Reply:
this works, but I was actually intentionally saying "Try Yourself" as a more direct call to action. But with "Nx" in the middle works too ofc